### ISSUE TITLE
change_post_status_data_type_to_posts

### ISSUE THREAD

**Issue by reifnotreef** [learn-co-curriculum/rails-generators-readme#19]
"We can simply add in the `change_column` method like this: `change_column :posts, :post_status, :string` and after running `rake db:migrate` our schema will be updated."

Changing the post_status to string makes the test fail.

**Comment by gj** [comment]
Easy enough to change it back :)

```ruby
class ChangePostStatusDataTypeAgainToPosts < ActiveRecord::Migration
  def change
    change_column :posts, :post_status, :boolean
  end
end
```

**Comment by aturkewi** [comment]
Resolved this one in PR #29 
Keeping it as a string so a post can have many statuses `"draft", "published", "review", ...`
